ProcessMaker/screen-builder
# Fix: The validations of a screen are lost when they are inside a nested screen

## Issue & Reproduction Steps
The validations of a screen are lost when they are initially hidden then by an user action it is displayed again inside a nested screen.

Expected behavior: 
When fields with validations are displayed by an user action inside a nested screen their validation rules should also displayed and enabled.

Actual behavior: 
When fields with validations are displayed by an user action inside a nested screen their validation rules are not being displayed nor enabled.

## Solution
- Refresh the Nested Validation Rules when required.

## How to Test

1. Import the attached screen
2. Open the Nested screen FOUR-7257
3. Preview the screen
4. Click in "enable controls" button
5. The validations should be enabled

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-7290

## Code Review Checklist
